routes/discordactions.js

```diff
@@ -17,6 +18,7 @@ const router = express.Router();
 router.post("/groups", authenticate, checkIsVerifiedDiscord, validateGroupRoleBody, createGroupRole);
 router.get("/groups", authenticate, checkIsVerifiedDiscord, getAllGroupRoles);
 router.post("/roles", authenticate, checkIsVerifiedDiscord, validateMemberRoleBody, addGroupRoleToMember);
+router.get("/user/group/roles", authenticate, checkIsVerifiedDiscord, getGroupsRoleId);
```

Comment:
please discuss with others and change route

Reply:
Shubham test coverage you  mean adding details regarding the test in the description ?

Reply:
Done